test.js

```diff
@@ -85,3 +85,45 @@ test('add tasks', t => {
 
 	t.is(list._tasks.length, 4);
 });
+
+test('skip tasks', t => {
+	t.plan(5);
+
+	const list = new Listr([
+		{
+			title: 'Task 1',
+			task: () => new Listr([
+				{title: 'Task 1.1', task: () => t.pass() || '1.1'},
```

Comment:
I had the `t.plan(5)` and the `t.pass()` to ensure that all the non-skipped tasks still ran. 

`t.pass()` doesn't return a value and currently the implementation of Listr requires the `task()` function return a result so that's why I returned *something* from each of the `task` functions.

But as mentioned in the PR description I'm keen to give the tests some love, this was just to prove the concept really.